docpipe/html.py

```diff
@@ -286,6 +287,24 @@ def __call__(self, context):
             br.getparent().addnext(p)
             br.getparent().remove(br)
 
+        # TODO: deal with e.g.
+        #  <p><b>[PCh1]</b><b>CHAPTER 1<br>THE INTERPRETATION OF LAWS ACT</b></p>
+        #  better
+        for br in reversed(list(context.html.xpath('.//p/b/br'))):
```

Comment:
So we're looking for `br` deep inside `p` tags and breaking those onto new lines. Is it worthwhile making this more generic, eg `i`? It'd have to re-create all the tags in the tree between the `p` and the `br`. I don't think we want to support arbitrary tags (we could be deep inside divs etc.)

I think we need a test for this.